spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -203,8 +205,6 @@ protected void doStop() {
 
 	private class ListenerConsumer implements SchedulingAwareRunnable {
 
-		private final Log logger = LogFactory.getLog(this.getClass());
-
 		private final CommitCallback callback = new CommitCallback();
```

Comment:
Yeah... I see.
What I don't like that I really fine with the `logger` without `this.`, but there is no `exclude` option for the `RequireThis` rule...

So, let's move it back and add `this.` because there is no choice! At least I did similar for the `KafkaTemplate` :smile: